build.sbt

```diff
@@ -65,3 +66,57 @@ libraryDependencies ++= Seq(
   "com.sksamuel.elastic4s" %% "elastic4s-http-streams" % s4sVersion,
   "com.sksamuel.elastic4s" %% "elastic4s-json-play" % s4sVersion
 )
+// Google app engine logging
+libraryDependencies += "com.google.cloud" % "google-cloud-logging-logback" % "0.120.0-alpha"
```

Comment:
@JarrodBaker we should remove this as it was causing problems in the deployment. the default logging system is enough to be capture by the underlying logging transport

Reply:
Good pickup! Thanks and fixed!